sass/_component_versionpicker_fix.scss

```diff
@@ -8,7 +8,7 @@ These overrides are necessary to make the version picker work with the Sphinx Wa
   bottom: 50px;
   right: 50px;
   max-width: $modal-sm;
-  max-height: calc(100vh - 50px);
+  height:80vh;
```

Comment:
Based on the comment https://github.com/wagtail/sphinx_wagtail_theme/issues/247#issuecomment-1429794795


I don't think this will fix it. 

You will need to ensure thaf this CSS override is more specific than the default styles. 

Maybe move this down and put it within `&.shift-up {`

Reply:
ohh sorry my bad Instead of adding it to `&.shift-up` I added the **height:80vh;** into `.rst-versions` I made the changes but I am little confused that should I make a new branch and then commit the changes and then make a PR or without making a new Branch I should just commit the changes and make a PR?

Reply:
Just push new changes to the same branch.
Here is a guide for new contributions that explains this and other helpful tips - https://github.com/wagtail/wagtail/issues/10070

Reply:
Pushed the changes in the same branch please look into it